tfx/dependencies.py

```diff
@@ -42,7 +40,7 @@ def make_required_install_packages():
       'pyyaml>=3.12,<6',
       'six>=1.10,<2',
       'tensorflow>=1.15.2,!=2.0.*,!=2.1.*,!=2.2.*,<3',
-      'tensorflow-data-validation>=0.23,<0.24',
+      'tensorflow-data-validation>=0.23.1,<0.24',
```

Comment:
Please resolve this comment when TFDV 0.23.1 is out.

Reply:
Acknowledged